interview-questions.md

```diff
@@ -8,36 +8,93 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: To fix the mistake, you can create a migration to add in the foriegn key column. 
 
-Researched answer:
+First in the Terminal, you want to run the command: rails migration add_column_cohort_id_to_students
+A migration file will be created. In the migration file you will want to add the following code: add_column :students :cohort_id :integer
+Once the file is saved, run the commad: rails db:migrate 
+This will update the schema file with the newly added column. Verify by checking schema file to see if column was created successfully. 
+
+Researched answer: While this alternative is not best practice, if you have not migrated your database yet. You can go into the schema file itself and add/edit any information for your model. Highly advised not to do this because in my opinion it defeats the purpose of migrations. Also, migrations are a log of all the changes you have made to your model. So, if you go through a lot of migrations and end up coming into problems. You can reference those migration files to check for any issues. 
```

Comment:
Updating the schema should never be done as that file should be considered a read only file.  Your schema fully relies on the migrations to have the correct build.  However, if you haven't migrated your database yet, you can still modify the initial migration file to add the column.